server/task/task-distributed/src/main/scala/org/apache/james/task/eventsourcing/cassandra/CassandraTaskExecutionDetailsProjectionDAO.scala

```diff
@@ -76,17 +84,26 @@ class CassandraTaskExecutionDetailsProjectionDAO(session: Session, typesProvider
     .executeRows(listStatement.bind())
     .map(readRow)
 
-  private def readRow(row: Row): TaskExecutionDetails = new TaskExecutionDetails(
-    taskId = TaskId.fromUUID(row.getUUID(TASK_ID)),
-    `type` = TaskType.of(row.getString(TYPE)),
-    status = TaskManager.Status.fromString(row.getString(STATUS)),
-    submittedDate = CassandraZonedDateTimeModule.fromUDT(row.getUDTValue(SUBMITTED_DATE)),
-    submittedNode = Hostname(row.getString(SUBMITTED_NODE)),
-    startedDate = CassandraZonedDateTimeModule.fromUDTOptional(row.getUDTValue(STARTED_DATE)),
-    ranNode = Optional.ofNullable(row.getString(RAN_NODE)).map(Hostname(_)),
-    completedDate = CassandraZonedDateTimeModule.fromUDTOptional(row.getUDTValue(COMPLETED_DATE)),
-    canceledDate = CassandraZonedDateTimeModule.fromUDTOptional(row.getUDTValue(CANCELED_DATE)),
-    cancelRequestedNode = Optional.ofNullable(row.getString(CANCEL_REQUESTED_NODE)).map(Hostname(_)),
-    failedDate = CassandraZonedDateTimeModule.fromUDTOptional(row.getUDTValue(FAILED_DATE)),
-    additionalInformation = Optional.empty)
+  private def readRow(row: Row): TaskExecutionDetails = {
+    val taskType = TaskType.of(row.getString(TYPE))
+    new TaskExecutionDetails(
+      taskId = TaskId.fromUUID(row.getUUID(TASK_ID)),
+      `type` = TaskType.of(row.getString(TYPE)),
```

Comment:
I think you can remove the ` character

Reply:
No, because in scala `type` is a reserved keyword.